### ISSUE TITLE
Stuck on multiplexing avc stream

### ISSUE THREAD

**Issue by Selur** [gpac/gpac#1190]
I'm trying to multiplex these files: 
https://drive.google.com/open?id=1XAYDoTzqO9jJFyCI9VcrIWEVNpwnM6gI
using (MP4Box - GPAC version 0.7.2-DEV-rev985-g6096388c4-ab-suite):
```
MP4Box -par 1=1:1 -add "E:\Temp\112_15_32_58_2410_02.264"#video:fps=25:delay=80 -brand avc1 -add "E:\Temp\iId_2_aid_4352_ms_15_32_58_2410_01.ac3"#audio:lang= -itags tool="Hybrid 2019.01.02.1" -tmp "E:\Temp" -new "E:\Output\15_32_58_2410__04.mp4"
```
I get:
```
[32mAVC-H264 import - frame size 1920 x 1080 at 25.000 FPS
[0mImporting AVC-H264: |                    | (01/100)
Importing AVC-H264: |                    | (02/100)
Importing AVC-H264: |                    | (03/100)
Importing AVC-H264: |                    | (04/100)
Importing AVC-H264: |=                   | (05/100)
Importing AVC-H264: |=                   | (06/100)
Importing AVC-H264: |=                   | (07/100)
Importing AVC-H264: |=                   | (08/100)
Importing AVC-H264: |=                   | (09/100)
Importing AVC-H264: |==                  | (10/100)
Importing AVC-H264: |==                  | (11/100)
Importing AVC-H264: |==                  | (12/100)
Importing AVC-H264: |==                  | (13/100)
Importing AVC-H264: |==                  | (14/100)
Importing AVC-H264: |===                 | (15/100)
Importing AVC-H264: |===                 | (16/100)
Importing AVC-H264: |===                 | (17/100)
Importing AVC-H264: |===                 | (18/100)
Importing AVC-H264: |===                 | (19/100)
Importing AVC-H264: |====                | (20/100)
Importing AVC-H264: |====                | (21/100)
Importing AVC-H264: |====                | (22/100)
Importing AVC-H264: |====                | (23/100)
Importing AVC-H264: |====                | (24/100)
Importing AVC-H264: |=====               | (25/100)
Importing AVC-H264: |=====               | (26/100)
Importing AVC-H264: |=====               | (27/100)
Importing AVC-H264: |=====               | (28/100)
Importing AVC-H264: |=====               | (29/100)
Importing AVC-H264: |======              | (30/100)
Importing AVC-H264: |======              | (31/100)
Importing AVC-H264: |======              | (32/100)
Importing AVC-H264: |======              | (33/100)
Importing AVC-H264: |======              | (34/100)
Importing AVC-H264: |=======             | (35/100)
Importing AVC-H264: |=======             | (36/100)
Importing AVC-H264: |=======             | (37/100)
Importing AVC-H264: |=======             | (38/100)
Importing AVC-H264: |=======             | (39/100)
Importing AVC-H264: |========            | (40/100)
Importing AVC-H264: |========            | (41/100)
Importing AVC-H264: |========            | (42/100)
Importing AVC-H264: |========            | (43/100)
Importing AVC-H264: |========            | (44/100)
Importing AVC-H264: |=========           | (45/100)
Importing AVC-H264: |=========           | (46/100)
Importing AVC-H264: |=========           | (47/100)
Importing AVC-H264: |=========           | (48/100)
Importing AVC-H264: |=========           | (49/100)
Importing AVC-H264: |==========          | (50/100)
Importing AVC-H264: |==========          | (51/100)
Importing AVC-H264: |==========          | (52/100)
Importing AVC-H264: |==========          | (53/100)
Importing AVC-H264: |==========          | (54/100)
Importing AVC-H264: |===========         | (55/100)
Importing AVC-H264: |===========         | (56/100)
Importing AVC-H264: |===========         | (57/100)
Importing AVC-H264: |===========         | (58/100)
Importing AVC-H264: |===========         | (59/100)
Importing AVC-H264: |============        | (60/100)
Importing AVC-H264: |============        | (61/100)
Importing AVC-H264: |============        | (62/100)
Importing AVC-H264: |============        | (63/100)
Importing AVC-H264: |============        | (64/100)
Importing AVC-H264: |=============       | (65/100)
Importing AVC-H264: |=============       | (66/100)
Importing AVC-H264: |=============       | (67/100)
Importing AVC-H264: |=============       | (68/100)
Importing AVC-H264: |=============       | (69/100)
Importing AVC-H264: |==============      | (70/100)
Importing AVC-H264: |==============      | (71/100)
Importing AVC-H264: |==============      | (72/100)
Importing AVC-H264: |==============      | (73/100)
Importing AVC-H264: |==============      | (74/100)
Importing AVC-H264: |===============     | (75/100)
Importing AVC-H264: |===============     | (76/100)
Importing AVC-H264: |===============     | (77/100)
Importing AVC-H264: |===============     | (78/100)
Importing AVC-H264: |===============     | (79/100)
Importing AVC-H264: |================    | (80/100)
Importing AVC-H264: |================    | (81/100)
Importing AVC-H264: |================    | (82/100)
Importing AVC-H264: |================    | (83/100)
Importing AVC-H264: |================    | (84/100)
Importing AVC-H264: |=================   | (85/100)
Importing AVC-H264: |=================   | (86/100)
Importing AVC-H264: |=================   | (87/100)
Importing AVC-H264: |=================   | (88/100)
Importing AVC-H264: |=================   | (89/100)
Importing AVC-H264: |==================  | (90/100)
Importing AVC-H264: |==================  | (91/100)
Importing AVC-H264: |==================  | (92/100)
Importing AVC-H264: |==================  | (93/100)
Importing AVC-H264: |==================  | (94/100)
Importing AVC-H264: |=================== | (95/100)
Importing AVC-H264: |=================== | (96/100)
Importing AVC-H264: |=================== | (97/100)
Importing AVC-H264: |=================== | (98/100)
Importing AVC-H264: |=================== | (99/100)
Importing AVC-H264: |====================| (100/100)
```
as output, but that's it MP4Box is stuck and never closes the 0000000000B76BB0_isotmp file inside the temp folder.
I first thought that the audio was the problem, but leaving out the audio doesn't fix the issue either.
Muxing the two streams using ffmpeg works fine. 

Cu Selur

**Comment by aureliendavid** [comment]
Apparently this has to do with the video stream having lots of samples with a negative CTS offset. 

In that case, at the end of the import, it gets stuck into an old block of code that start with a comment saying `/* this should never happen */`  so that's not great (https://github.com/gpac/gpac/blob/master/src/media_tools/media_import.c#L5682)

ffmpeg emits warnings of these samples but does manages to produce a correct output, so that's doable

I'll look further into what we should do in this case.

**Comment by jeanlf** [comment]
this should be fixed in GPAC 0.9.0 DEV
